deploying_django.md

```diff
@@ -82,25 +82,25 @@ So now your directory should look like this:
     ├── app3
     └── .gitignore
 ```
-> Note you can choose to add a file called `.ebignore` in which you can type in all the files and folders that you don't want to be uploaded when using command line interface to upload. If you have `.gitignore` you won't require `.ebignore`, but if you do add `.ebignore` your `.gitignore` won't be read and only the files and directories specifed in `.ebignore` will be ignored.
+> Note you can choose to add a file called `.ebignore` in which you can type in all the files and folders that you don't want to be uploaded when using the command-line interface to upload. If you have `.gitignore` you won't require `.ebignore`, but if you do add `.ebignore` your `.gitignore` won't be read and only the files and directories specified in `.ebignore` will be ignored.
 
-## 1. Creating application using the aws console.
-This is one of the easiest way to set up your django project up and running.
+## 1. Creating an application using the AWS console.
+This is one of the easiest ways to set up your Django project and run it.
```

Comment:
"Django project and run it" can you change this back to "This is one of the easiest way to set up your Django project up and running."